### ISSUE TITLE
Setup Quality tools

### ISSUE THREAD

**Issue by eyedol** [ushahidi/SMSSync#285]
Setup static analytic tools to catch obvious bugs.

See for a sample blog post; http://vincentbrison.com/2014/07/19/how-to-improve-quality-and-syntax-of-your-android-code/

**Comment by alxndrsn** [comment]
Hi Henry,
This looks like easy work, but it would be good to get a list of tools to integrate.  And maybe each could have its own issue, so they can be checked off as they're set up.
What do you think?

**Comment by eyedol** [comment]
Agreed. Updated the description with checkboxes and tools to be integrated

**Comment by alxndrsn** [comment]
I don't see the update o_O

**Comment by eyedol** [comment]
(headbang) I didn't hit `Update comment` Check again.
